### DISCUSSION TITLE
rendering color?

### DISCUSSION THREAD

**Discussion by DoctorSubtilis** [Zettlr/Zettlr#3471] [Ideas]
There is a way to rend a text colored (<span style="color: blue"> is shown as _code_, not rendered)?   
Maybe this could be a new feature?

**Comment by kyaso** [comment]
Colored text is not part of Markdown. So, Zettlr does not support rendering colored text.

However, as pointed out in this Stackoverflow [answer](https://stackoverflow.com/a/35485694/3727722), you can use HTML to get colored text. Note that you'll have to export the document in order to see the colored text.

**Comment by wheelercj** [comment]
I guess this is probably not what you mean, but just in case:

Since Zettlr can instantly display some LaTeX elements, you can use colors like this:

```
$\color{red} hi$
$\rm\color{red} hi$
$\colorbox{red}{hi}$
```

![image](https://user-images.githubusercontent.com/39012918/169712412-36cd6f59-a0d2-4696-aa36-f585425a32fc.png)

You could make these easier to use by adding autocorrect options to Zettlr's settings like this:

![image](https://user-images.githubusercontent.com/39012918/169712476-fc2dada6-d2be-41d0-b702-30038c70432f.png)

**Reply by ghost** [reply]
Hey, thank you, this is something I was looking for! Do you know of a way to do this that leaves spaces between words in the preview? I need this 'feature' to highlight in red the parts of a text that I'm writing that I don't like and want to review later! 
<img width="531" alt="image" src="https://user-images.githubusercontent.com/130041188/230314674-fad0bc42-ef3e-4490-b7a8-f2aba5a5fb91.png">

**Reply by wheelercj** [reply]
Hi, glad to help! Right now the only way I know is to put a backslash before each space. I feel like there should be an easier way but I searched for a bit and didn't find an answer.

**Reply by UFOMelkor** [reply]
That occurs because `$` starts the math mode in LaTeX. You can use `\text{}` to go back into text mode: `I'm writing something $\color{red}\text{in red}$.`

![image](https://user-images.githubusercontent.com/571106/232272291-b1c5b176-9e49-404a-b226-073bbcd11304.png)

**Reply by ghost** [reply]
Yeah that makes sense! I ended up using == to highlight in yellow as that seems to be the quickest solution when writing text.
